gulpfile.js

```diff
@@ -42,24 +45,18 @@ gulp.task('build', cb => {
 gulp.task('lint', ['lint:styles', 'lint:scripts', 'lint:tests'])
 
 // Task to run the tests
-// This runs preview first, to copy assets from dist/bundle to /public, then runs the tests
+// This runs build before testing the preview task, to copy assets to dist/bundle and /public
 gulp.task('test', cb => {
-  runSequence('lint', 'preview', 'test:lib', 'test:toolkit', 'test:preview', cb)
+  runSequence('lint', 'test:lib', 'test:toolkit', 'build', 'test:preview', cb)
 })
 
 // Package the contents of dist
 gulp.task('package', cb => {
   runSequence('build', ['package:gem', 'package:npm'], cb)
 })
 
-// Start server
-// This runs the preview task first then starts the server
-gulp.task('start', cb => {
-  runSequence('preview', ['start:server'], cb)
-})
-
-// Copy files to /public
-// This runs the build task first, then copies the assets from dist/bundle to /public
+// Preview
+// This runs the build task first, watches and starts the server
 gulp.task('preview', cb => {
-  runSequence('build', ['preview:copy:styles', 'preview:copy:images', 'preview:copy:js'], cb)
+  runSequence('build', ['browser-sync', 'watch', 'start:server'], cb)
```

Comment:
An alternative would be to start browser sync in the same task as nodemon({}).
https://www.npmjs.com/package/gulp-nodemon#using-gulp-nodemon-with-browser-sync